server/utils/utils.go

```diff
@@ -60,3 +61,8 @@ func LoadSession(mm *pluginapi.Client, sessionID, actingUserID string) (*model.S
 	}
 	return session, nil
 }
+
+func DumpObject(c interface{}) {
```

Comment:
Is this one for future debugging sessions?

Reply:
Yes. It was not my intention to commit this change, it is something I was using locally.

Do you want me to revert this change?

Reply:
Might be useful for others as well (and avoids the need for everyone to constantly add and remove pre-checkin similar methods). Maybe just add a comment re: its purpose.